packages/angular/src/jsonforms-root.component.ts

```diff
@@ -93,6 +96,12 @@ export class JsonForms implements OnChanges, OnInit {
         const newAjv = changes.ajv;
         const newConfig = changes.config;
 
+        // if only data changed and we just emitted, don't do anything
+        if (Object.keys(changes).length === 1 && newData && this.emittedChange) {
+            this.emittedChange = false;
+            return;
+        }
```

Comment:
the check is only triggered when an updated was running, on the second run which is triggered because angular sets the same data back in:
change -> updateCore -> emit due to 2way binding -> update data due to 2 way binding -> call updateCore again
I wanted to get rid of the additional updateCore, but if you say this is a noop, we can remove this